### ISSUE TITLE
[FR] Cache maintenance, deletion

### ISSUE THREAD

**Issue by Artanicus** [chubin/wttr.in#170]
Issue:
At current the weather cache doesn't seem to get cleared at any point and just keeps building up. With a slow query volume (~1 hit per 5 minutes from a dashboard TV) for ~year my cache for a single lat,long pair is ~300MB. While not huge, it's still something I'd rather not lug around or manually clean.

Proposed solution:
Expire cache entries as soon as a new replacement cache entry is created, or expire them after a set amount of time. As far as I understand there is no way to even query for historical data from the UI so cache expiration shouldn't affect any end-users and only ease administration.

**Comment by chubin** [comment]
You are right, it is not possible at the moment (to query historical data), but it is planned.
The idea is to query it using the @2018-10-10 notation (as it is for /Moon now).
On the other hand, you can't use the cache for that, because the cache is only cache.

Cache is cleaned up using a cron script (and it is not yet on github).
